Quadriga/src/main/java/edu/asu/spring/quadriga/domain/impl/projectblog/ProjectBlogEntry.java

```diff
@@ -113,4 +115,34 @@ public void setAuthor(IUser author) {
         this.author = author;
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public String getStripped(String input, int wordCount) {
+        
+        String[] words = input.split(" ");
+        
+        if(words.length < wordCount)
+            return input;
+        
+        String result = "";
```

Comment:
Don't use Strings here. Please read this:
http://www.instanceofjava.com/2015/01/string-vs-stringbuffer-vs-stringbuilder.html
http://java.globinch.com/java-quick-tips/performance-string-stringbuffer-stringbuilder-memory-runtime-analysis/